packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -6,33 +6,17 @@ import 'package:flutter/foundation.dart';
 
 import 'framework.dart';
 
-/// A [ValueKey] that defines where [PageStorage] values will be saved.
+/// A key can be used to persist the widget state in storage after
+/// the destruction and will be restored when recreated.
 ///
-/// [Scrollable]s ([ScrollPosition]s really) use [PageStorage] to save their
-/// scroll offset. Each time a scroll completes, the scrollable's page
-/// storage is updated.
+/// Each key with its value plus the ancestor chain of other PageStorageKeys need to be unique,
+/// i.e. the widget's closest ancestor [PageStorage]. To make it possible for a
```

Comment:
This sentence seems to be incomplete?

Do you mean: "... need to be unique within the widget's closest ancestor [PageStorage]."?

Reply:
This comment is still unaddressed.